code/modules/research/designs/autolathe_desings/autolathe_designs_medical_and_dinnerware.dm

```diff
@@ -214,3 +215,13 @@
 	build_path = /obj/item/reagent_containers/glass/bottle/vial/large
 	category = list("initial","Medical","Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
+
+
+/datum/design/hypospray/mkii
+	name = "Hypospray Mk. II"
+	id = "hypospray_mkii"
+	build_type = PROTOLATHE | NO_PUBLIC_LATHE
+	materials = list(/datum/material/iron = 1600, /datum/material/glass = 1000)
+	build_path = /obj/item/hypospray/mkii
+	category = list("initial", "Medical","Medical Designs")
+	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL
```

Comment:
This entire part should be moved to ../code/modules/research/designs/medical_designs.dm, due to being linked to the techweb & protolathe as opposed to the autolathe,

Reply:
Just the 'moving the hypo design from where you placed it to the medical_designs file' left and this should be all good.

Reply:
Suggestions implemented!